apps/crn-server/src/data-providers/users.data-provider.ts

```diff
@@ -59,6 +56,55 @@ export class UserSquidexDataProvider implements UserDataProvider {
     return parseGraphQLUserToDataObject(findUsersContent);
   }
 
+  async create(userCreateDataObject: UserCreateDataObject): Promise<string> {
+    const { teams, labIds, ...input } = userCreateDataObject;
+
+    const user = cleanUser(input);
+
+    const inputUser: InputUser['data'] = {
+      ...user,
+      email: {
+        iv: input.email,
+      },
+      firstName: {
+        iv: input.firstName,
+      },
+      lastName: {
+        iv: input.lastName,
+      },
+      role: {
+        iv: input.role,
+      },
+      teams: {
+        iv: teams?.map((team) => ({ id: [team.id], role: team.role })) || [],
+      },
+      labs: {
+        iv: labIds,
+      },
+      connections: {
+        iv: [],
+      },
+      onboarded: {
+        iv: input.onboarded || false,
+      },
+      avatar: {
+        iv: (input.avatar && [input.avatar]) || null,
+      },
+      degree: {
+        iv: input.degree || null,
+      },
+      questions: {
+        iv: input.questions?.map((question) => ({ question })) || null,
+      },
+      social: {
+        iv: (input.social && [{ ...input.social }]) || [],
+      },
+    };
+    const { id } = await this.userSquidexRestClient.create(inputUser);
```

Comment:
Should this return only the id or the newly created user?

I might be wrong but in other places we've returned the newly created entity.

Reply:
Pretty sure we return just the string everywhere else?
https://github.com/yldio/asap-hub/blob/b424a7df38b188a6b189bd7a5cb2c6dd16a42f70/apps/crn-server/src/data-providers/assets.data-provider.ts#L7
https://github.com/yldio/asap-hub/blob/b424a7df38b188a6b189bd7a5cb2c6dd16a42f70/apps/crn-server/src/data-providers/external-authors.data-provider.ts#L5
https://github.com/yldio/asap-hub/blob/b424a7df38b188a6b189bd7a5cb2c6dd16a42f70/apps/crn-server/src/data-providers/research-outputs.data-provider.ts#L38